server/app/views/admin/programs/ProgramApplicationView.java

```diff
@@ -113,14 +109,13 @@ private Tag renderApplicationBlock(long programId, Block block, Collection<Answe
             Styles.MB_4);
   }
 
-  private Tag renderAnswer(long programId, AnswerData answerData) {
+  private Tag renderAnswer(AnswerData answerData) {
     LocalDate date =
         Instant.ofEpochMilli(answerData.timestamp()).atZone(ZoneId.systemDefault()).toLocalDate();
     Tag answerContent;
     if (answerData.fileKey().isPresent()) {
       String encodedFileKey = URLEncoder.encode(answerData.fileKey().get(), StandardCharsets.UTF_8);
-      String fileLink =
-          controllers.routes.FileController.adminShow(programId, encodedFileKey).url();
+      String fileLink = controllers.routes.FileController.acledAdminShow(encodedFileKey).url();
```

Comment:
How will these work for legacy files?

Reply:
+1

Reply:
Yeah, thank you. Coming back to this and realizing we need to migrate the old file data to have ACLs before we can begin issuing the new URLs